### PR TITLE
mind entry points when structuring the manifest

### PR DESCRIPTION
**Pull request by roymiloh**
This PR changes the structure of the emitted manifest to the following one:

```ts
interface Manifest {
  entry: string[];
}
```

The idea is to exclusively select files of specific entry, if needed (for example - in order to render only one specific bundle files in html). Currently all chunks are on the top level (flatten), so it's impossible to know which chunk belongs to each entry point, in case there are multiple.

### CODE REVIEW

File: test/typescript/features/manifest/package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "feature",
+  "private": true,
+  "yoshi": {
+    "projectType": "app",
+    "splitChunks": {
```
Comment (ronami) [inline test/typescript/features/manifest/package.json]: I think we can configure this with `chunks: "all"` so it has less clutter (https://webpack.js.org/plugins/split-chunks-plugin/#splitchunkschunks)

File: test/typescript/features/manifest/package.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      "maxSize": 100,
+      "minSize": 100
+    },
+    "entry": {
+      "app1": "./client.tsx",
```
Comment (ronami) [inline test/typescript/features/manifest/package.json]: Having the same name both for entries will be less confusing

File: test/typescript/features/manifest/pom.xml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ronami) [inline test/typescript/features/manifest/pom.xml]: Is this needed? Can this be moved to common template?

File: test/typescript/features/manifest/src/some-module.ts [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import styles from './styles.css';
+
+export const a = 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa'
```
Comment (ronami) [inline test/typescript/features/manifest/src/some-module.ts]: Can be `const a = "a"` if we use the suggestion above about `chunks: "all"`

File: test/typescript/features/manifest/manifest.test.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
+const path = require('path');
```
Comment (yanivefraim) [inline test/typescript/features/manifest/manifest.test.js]: we have all tests in JS and TS. We should have this one in JS as well...

File: test/typescript/features/manifest/pom.xml (hunk 3 above)
Comment (roymiloh) [reply test/typescript/features/manifest/pom.xml]: In order to generate url in prod manifest which contains parastorage thingy, it requires artifact id.

**Pull request closed without merging**